desktop/core/src/desktop/api_public_urls_v1.py

```diff
@@ -39,7 +39,9 @@
   re_path(r'^banners/?$', api_public.get_banners, name='core_banners'),
   re_path(r'^get_config/?$', api_public.get_config),
   re_path(r'^get_namespaces/(?P<interface>[\w\-]+)/?$', api_public.get_context_namespaces),  # To remove
+]
 
+urlpatterns += [
```

Comment:
why this `urlpatterns +=`

Reply:
Did this to not let linter messup with url mapping order for better readability.